python/langsmith/pytest_plugin.py

```diff
@@ -75,6 +76,7 @@ def pytest_runtest_call(item):
         request_obj = getattr(item, "_request", None)
         if request_obj is not None and "request" not in item.funcargs:
             item.funcargs["request"] = request_obj
+        if request_obj is not None and "request" not in item._fixtureinfo.argnames:
```

Comment:
@baskaryan checking this separately seems to have fixed the problem

I am not sure why we are modifying internals like this